mmv1/third_party/terraform/services/container/node_config.go.erb

```diff
@@ -597,6 +609,13 @@ func schemaNodeConfig() *schema.Schema {
 								Optional: true,
 								Description: `Set the CPU CFS quota period value 'cpu.cfs_period_us'.`,
 							},
+							"insecure_kubelet_readonly_port_enabled": {
```

Comment:
I went back and forth on this a bit. This setting is used in a bunch of different places, and I wasn't totally sure that the description should be the same for all of them?

At the time, I think it may not have been 100% obvious whether the description should be ultimately the same, as well as whether the `Optional`, `Computed`, etc. settings would be the same. I can update it to use that.

Reply:
sounds good, thanks!